### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by MelbourneDeveloper**
I'm not sure if "stand up" is a phrasal verb I'm not familiar with in this context, but "set up" seems like the right choice to me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Octopus People
+﻿# Octopus People
```
Comment (MelbourneDeveloper) [inline README.md]: Should this be "as Octopus", or "at Octopus"?

Comment (michaelnoonan) [reply README.md]: I agree: "at Octopus". :)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-This repository describes the culture we strive for as Octopus, recognising our culture is the sum product of our people, the way we behave towards each other, and the behaviours we tolerate.
+This repository describes the culture we strive for as Octopus. It recognises our culture is the sum product of our people, the way we behave towards each other, and the behaviours we tolerate.
```
Comment (wordlee) [inline README.md]:
```suggestion
This repository describes the culture we strive for at Octopus. It recognises our culture is the sum product of our people, the way we behave towards each other, and the behaviours we tolerate.
```

**Comment by MelbourneDeveloper** [comment]
@michaelnoonan all the changes seem to be approved, but closing the pull request did not seem to commit the changes to the master branch.

**Comment by michaelnoonan** [comment]
We had to merge your PR. All done. Thanks for contributing!

**Pull request merged**